### DISCUSSION TITLE
WalletConnect v2 ?

### DISCUSSION THREAD

**Discussion by callmejm** [wevm/wagmi#1183] [Q&A]
Hi first for all, glade to have wagmi. Is wagmi currently support walletconnect v2 ? soon v1 will [shut down on March 2023 ](https://medium.com/walletconnect/walletconnect-v1-0-sunset-notice-and-migration-schedule-8af9d3720d2e)

**Comment by tmm** [comment]
We are working with the WalletConnect team on this in https://github.com/wagmi-dev/wagmi/pull/1208
